frontend_lib/src/component/EmojiReactions/EmojiReactionButtons.jsx

```diff
@@ -0,0 +1,252 @@
+import 'emoji-mart/css/emoji-mart.css'
+import emojiData from 'emoji-mart/data/all.json'
+import { getEmojiDataFromNative} from 'emoji-mart'
+import { Picker } from 'emoji-mart'
+import { UncontrolledPopover, PopoverBody } from 'reactstrap'
+
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+import { andList } from '../../helper.js'
+import IconButton from '../Button/IconButton.jsx'
+
+class EmojiPickerButton extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = { pickerDisplayed: false }
+    this.buttonRef = React.createRef()
+  }
+
+  handleSelect = (emoji) => {
+    this.props.onAddReaction(emoji.native)
+    this.setState({ pickerDisplayed: false })
+  }
+
+  handleBodyClick = (e) => {
+    if (!this.state.pickerDisplayed) return
+
+    let parent = e.target
+    while (parent) {
+      if (parent.classList && parent.classList.contains('EmojiPickerPopover')) {
+        return
+      }
+      parent = parent.parentNode
+    }
+
+    this.setState({ pickerDisplayed: false })
+    e.preventDefault()
+    e.stopPropagation()
+  }
+
+  componentDidMount () {
+    document.body.addEventListener('click', this.handleBodyClick)
+    document.body.addEventListener('touch', this.handleBodyClick)
+  }
+
+  componentWillUnmount () {
+    document.body.removeEventListener('click', this.handleBodyClick)
+    document.body.addEventListener('touch', this.handleBodyClick)
+  }
+
+  render () {
+    const { props } = this
+
+    const emojiMartI18n = {
+      search: props.t('Search'),
+      clear: props.t('Clear'),
+      notfound: props.t('No Emoji Found'),
+      skintext: props.t('Choose your default skin tone'),
+      categories: {
+        search: props.t('Search Results'),
+        recent: props.t('Frequently Used'),
+        smileys: props.t('Smileys & Emotion'),
+        people: props.t('People & Body'),
+        nature: props.t('Animals & Nature'),
+        foods: props.t('Food & Drink'),
+        activity: props.t('Activity'),
+        places: props.t('Travel & Places'),
+        objects: props.t('Objects'),
+        symbols: props.t('Symbols'),
+        flags: props.t('Flags'),
+        custom: props.t('Custom')
+      },
+      categorieslabel: props.t('Emoji categories'),
+      skintones: {
+        1: props.t('Default Skin Tone'),
+        2: props.t('Light Skin Tone'),
+        3: props.t('Medium-Light Skin Tone'),
+        4: props.t('Medium Skin Tone'),
+        5: props.t('Medium-Dark Skin Tone'),
+        6: props.t('Dark Skin Tone')
+      }
+    }
+
+    return (
+      <>
+        <button
+          title={props.t('React')}
+          ref={this.buttonRef}
+          className={classnames('ReactionButton__buttonpicker', { active: this.state.pickerDisplayed })}
+          onClick={() => this.setState(
+            prev => ({
+              pickerDisplayed: !prev.pickerDisplayed
+            })
+          )}
+        >
+          <i className='far fa-smile' />
+        </button>
```

Comment:
I tried this, but since I don't use IconButton for the other buttons, using an IconButton there leads to visual inconsistencies (colors, borders, size) and I would need to manually revert the IconButton's style and this is prone to break as IconButton evolves.

However, if we use IconButton for the other buttons, we should definitively  use it there too.